pcsx2/gui/Dialogs/GSDumpDialog.cpp

```diff
@@ -0,0 +1,784 @@
+/*  PCSX2 - PS2 Emulator for PCs
+ *  Copyright (C) 2002-2010  PCSX2 Dev Team
```

Comment:
2002-2010 ?

Reply:
don't we all prefer those simpler times?